Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec unicorn -c ./config/unicorn.rb -p ${PORT:-3107}
```

Comment:
Since this won't be sharing a VM with a bunch of other apps, the default port doesn't need to be unique like in govuk-puppet and can be something less surprising, like 8080 or 8081.